### PR TITLE
Refactor FXIOS-6989 [v119] Replace "Firefox account" with agnostic "Account" branding

### PR DESCRIPTION
**Pull request by PARAIPAN9**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-6989)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/TODO)

## :bulb: Description

- Replace "Firefox account" with agnostic "Account" branding
## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

**Review by lmarceau — COMMENTED** [review]
Hello! We had a discussion about how we can modify string in a recent eng meeting and we can't directly change the key anymore. We should create a new string for that string version until the old string is released in the last version it was put in (aka the last version it was used in is in the app store). 

The reason is that removing a string removes the string from l10n, which then removes the string from the version to be released when the import string script is run towards the release branch. Full explanation is under the wiki [here](https://github.com/mozilla-mobile/firefox-ios/wiki/How-to-add-and-modify-Strings)

Let me know if the explanation isn't clear, this is still a fairly new procedure

**Review by lmarceau — DISMISSED** [review]
as per comment

File: Client/Frontend/Strings.swift [hunk 1]
```diff
@@ -1636,10 +1636,10 @@ extension String {
 
         public struct Sync {
             public static let SyncTabs = MZLocalizedString(
-                key: "TabsTray.SyncTabs.SyncTabsButton.Title.v109",
+                key: "TabsTray.SyncTabs.SyncTabsButton.Title.v119",
```
Comment (razvanlitianu) [inline Client/Frontend/Strings.swift:1639]: I might be wrong but I guess we're going for v120 already

**Review by lmarceau — COMMENTED** [review]
Well done! One nit comment

**Review by adudenamedruby — APPROVED** [review]
LGTM, but I'll let @lmarceau also approve as she made comments before

**Review by lmarceau — APPROVED** [review]

**Pull request merged**